tests/eclient/testdata/profile.txt

```diff
@@ -24,9 +24,9 @@ eden pod deploy -n local-manager --memory=512MB docker://lfedge/eden-eclient:d9e
 
 test eden.app.test -test.v -timewait 10m RUNNING local-manager
 
-eden pod deploy -n app-profile-1 --memory=512MB docker://lfedge/eden-eclient:d9eb23f --profile=profile-1
-eden pod deploy -n app-profile-2 --memory=512MB docker://lfedge/eden-eclient:d9eb23f --profile=profile-2
-eden pod deploy -n app-profile-1-2 --memory=512MB docker://lfedge/eden-eclient:d9eb23f --profile=profile-1 --profile=profile-2
+eden pod deploy -n app-profile-1 --memory=512MB docker://milan4zededa/eden-eclient --profile=profile-1
```

Comment:
I take that it is `milan4zededa` only because it is a draft? We need this in a proper org on Docker HUB

Reply:
yes this is only temporary